aptos-move/e2e-move-tests/src/tests/transaction_context.rs

```diff
@@ -0,0 +1,265 @@
+// Copyright © Aptos Foundation
```

Comment:
Add error cases:
1. Optional user tx context is not available
2. One or all of these native functions error out

Reply:
The error cases are tested as unit tests because user transaction contexts are not available in unit testing.
https://github.com/aptos-labs/aptos-core/pull/11843/files#diff-8e778d93549cd2880d429daa01a9cc8b982085d3e7d81d11c0943f9e564e029dR119